source/jormungandr/tests/tests_mechanism.py

```diff
@@ -223,7 +223,7 @@ def setup_class(cls):
             instance = i_manager.instances[name]
             try:
                 retrying.Retrying(
-                    stop_max_delay=5000, wait_fixed=10, retry_on_result=lambda x: not instance.is_initialized
+                    stop_max_attempt_number=5, retry_on_result=lambda x: not instance.is_initialized
```

Comment:
Shouldn't there be a "wait_fixed" parameter also? As it is here, the attempts will be consecutive with no wait between each one?